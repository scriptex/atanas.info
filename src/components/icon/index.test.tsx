import * as React from 'react';
import { render } from '@testing-library/react';

import { Icon } from '.';

describe('Icon component', () => {
	it('Should render the Icon component', () => {
		const { asFragment } = render(<Icon name="test" className="test" />);

		expect(asFragment()).toMatchSnapshot();
	});
});
