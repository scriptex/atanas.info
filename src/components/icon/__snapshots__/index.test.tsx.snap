// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Icon component Should render the Icon component 1`] = `
<DocumentFragment>
  <svg
    class="test"
  >
    <use
      xlink:href="#test"
    />
  </svg>
</DocumentFragment>
`;
